scripts/aggregate-themes/theme-aggregator.js

```diff
@@ -139,14 +72,16 @@ class ThemeAggregator {
     }
 
     const assets = [];
-    let asset;
     themesToAggregate.forEach((theme) => {
-      asset = ThemeAggregator.aggregateTheme(theme, options, defaultFlag);
+      const asset = ThemeAggregator.triggerAggregationAndGeneration(theme, options, defaultFlag);
```

Comment:
Why the defaultFlag? Could we pass the boolean like: 
```
+ const isRootTheme = theme === defaultTheme;
+ const asset = ThemeAggregator.triggerAggregationAndGeneration(theme, options, isRootTheme);
- const asset = ThemeAggregator.triggerAggregationAndGeneration(theme, options, defaultFlag);
```

Reply:
Ok, the defaultFlag, isRoot, and isScoped flags I use are just...bad 😓 . Here's my shot at refactoring. Use a single flag to indicate if a theme is root. If that's false, we know it's scoped.

https://github.com/cerner/terra-toolkit/pull/376/commits/1d1bb30aefe21cc8976ec3b164fd57947e4299bb#diff-1672fd55492cdce050f5aa314b17029bR72